image-loader/app/model/ImageUpload.scala

```diff
@@ -169,7 +174,7 @@ class ImageUploadOps(store: ImageLoaderStore, config: ImageLoaderConfig, imageOp
             None
 
           baseImage = ImageUpload.createImage(uploadRequest, sourceAsset, thumbAsset, pngAsset, fullFileMetadata, cleanMetadata)
-          processedImage = SupplierProcessors.process(baseImage)
+          processedImage = new SupplierProcessors(metaDataConfig).process(baseImage)
```

Comment:
We probably don't want to new this up for every upload - can we inject a `SupplierProcessor` in?

Reply:
Have just pushed some changes to inject this in, please let me know what you think (i may be doing some scala no-no's!)